platform/android/export/export.cpp

```diff
@@ -1341,6 +1356,34 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			return str;
 		}
 	}
+
+	void _fix_godot_project_string(const Ref<EditorExportPreset> &p_preset) {
+		const char *locales[] = {
+			"ar", "bg", "ca", "cs", "da", "de", "el", "en", "es", "fa", "fi", "fr", "hi", "hr",
+			"hu", "in", "it", "iw", "ja", "ko", "lt", "lv", "nb", "nl", "pl", "pt", "ro", "ru",
+			"sk", "sl", "sr", "sv", "th", "tl", "tr", "uk", "vi", "zh", "zh_HK", "zh_TW"
+		};
+		String package_name = p_preset->get("package/name");
+		String project_name = get_project_name(package_name);
+		size_t num_locales = sizeof(locales)/sizeof(locales[0]);
+		//stores the default strings.xml
+		String xml_string = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n<resources>\n\t<string name=\"godot_project_name_string\">" + project_name + "</string>\n</resources>";
+		String directory = "res://android/build/res/values/strings.xml";
+		store_string_at_path(directory, xml_string);
+
+		for(int i = 1; i < num_locales; i++){
+			String lang = locales[i];
+			String prop = "application/config/name_" + lang;
+			if (ProjectSettings::get_singleton()->has_setting(prop)) {
+				String lang_project_name = ProjectSettings::get_singleton()->get(prop);
+				String xml_string = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n<resources>\n\t<string name=\"godot_project_name_string\">" + lang_project_name + "</string>\n</resources>";
+				String hyphen_lang = lang.replace("_","-r");
+				String directory = "res://android/build/res/values-"+hyphen_lang+"/strings"+hyphen_lang+".xml";
```

Comment:
Seems a bit risky to create new vars with the same name as existing vars in the outer scope